acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -72,7 +71,7 @@ public final void commit(Transaction t) {
 
         boolean success = false;
         try {
-            doCommit(t);
+            newObjectCommitted = doCommit(t);
             success = true;
         } catch (IOException e) {
             LOG.log(Level.SEVERE, "IOException while committing Committable: " + this.toString(), e);
```

Comment:
From the user's point of view, we just silently lose their work. It is probably preferable to actually crash.